### PR TITLE
RA-1752: Fix invalid username / password message

### PR DESCRIPTION
**Pull request by HerbertYiga**
Ticket ID:https://issues.openmrs.org/browse/RA-1752

Description
Added a flagged error message on the login page when the user signs in  with wrong credentials.The ticket ID contains the screen shots

### CODE REVIEW

File: omod/src/main/webapp/fragments/infoAndErrorMessages.gsp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <% if (errorMessage) { %>
-    <div id="error-message">${ errorMessage }</div>
+    <div class="alert alert-danger">
```
Comment (dkayiwa) [inline omod/src/main/webapp/fragments/infoAndErrorMessages.gsp:2]: What happens when you remove the div on line 2?

Comment (HerbertYiga) [reply omod/src/main/webapp/fragments/infoAndErrorMessages.gsp:2]: @dkayiwa dropping div 2 drops the  red back- grounded section in which we have the error message,should we go by that?,thanks

Comment (ibacher) [reply omod/src/main/webapp/fragments/infoAndErrorMessages.gsp:2]: @HerbertYiga So what if we had this instead:

```suggestion
    <div id="error-message" class="alert alert-danger" role="alert"><i class="icon-exclamantion-sign"></i>${ errorMessage }</div>
```

Comment (HerbertYiga) [reply omod/src/main/webapp/fragments/infoAndErrorMessages.gsp:2]: thanks @ibacher  let me try that and drop the screen shot on the ticket Id

Comment (HerbertYiga) [reply omod/src/main/webapp/fragments/infoAndErrorMessages.gsp]: @dkayiwa  @ibacher  made changes on this as suggested by @ibacher ,added a screen shot on the ticket Id named (latest screen shot) and this appears as the initial ones.thanks

Comment (HerbertYiga) [reply omod/src/main/webapp/fragments/infoAndErrorMessages.gsp]: @ibacher  hi ian,does this look good for merging?

**Pull request merged**